docs/router.md

```diff
@@ -0,0 +1,204 @@
+# About Routers
+
+`gpii.express.router` is a subclass of [`gpii.express.middleware`](middleware.md) that wraps
+[the  Express `router` object](http://expressjs.com/en/4x/api.html#router), which is simply a container for a collection
```

Comment:
Need to explain what a router actually does - "simply a container for a collection of middleware" is a bit blase : P That is, need to mention our recently clarified concept of a router which is something that "provides routing based purely on information present in the URL path"